docs/getting_started/examples.rst

```diff
@@ -10,8 +10,7 @@ Usage Examples
 The following notebooks demonstrate how Modin can be used for scalable data science:
 
 - Quickstart Guide to Modin [`Source <https://github.com/modin-project/modin/tree/master/examples/quickstart.ipynb>`_]
-- Using Modin with the NYC Taxi Dataset [`Source <https://github.com/modin-project/modin/blob/master/examples/jupyter/NYC_Taxi.ipynb>`_]
-- Using Experimental Modin in the cloud with the NYC Taxi Dataset on a AWS cluster [`Source <https://github.com/modin-project/modin/blob/master/examples/jupyter/NYC_Taxi_cloud.ipynb>`_]
```

Comment:
This should be removed in https://modin.readthedocs.io/en/stable/usage_guide/examples/index.html#modin-in-the-cloud too.